.github/PklProject.deps.json

```diff
@@ -0,0 +1,10 @@
+{
+  "schemaVersion": 1,
+  "resolvedDependencies": {
+    "package://pkg.pkl-lang.org/github.com/stefma/pkl-gha/com.github.action@0": {
+      "type": "local",
+      "uri": "projectpackage://pkg.pkl-lang.org/github.com/stefma/pkl-gha/com.github.action@0.0.5",
+      "path": "../.temp-pkl-deps/pkl-gha"
```

Comment:
This is using @StefMa's excellent pkl-gha, but this PR depends on at least version 0.0.5 (which isn't published). For now, in order to build the stuff here, you need to clone the repo down into `.temp-pkl-deps/pkl-gha`.

I'm planning on changing this to the remote version once that's available.